raiden/storage/sqlite.py

```diff
@@ -73,6 +74,13 @@ def _run_updates(self):
         )
         self.conn.commit()
 
+    def _log_raiden_run(self):
```

Comment:
I usually avoid these because they tend to develop into a badly written ORM over time due time questions like "How many rows do I want", "In which order would I like those rows", "After adding another column, some callers expect the old results while others need the new column included". But maybe I just had bad luck in the past.

If we want to do it, I suggest
```py
def get_last_runs(self, limit=10):
    """ Returns the latest runs (newest first, up to ``limit`` rows).

    Rows are sqlite3.Row instances which can be used like tuples or dictionaries.
   """
   ...
```

What I usually did instead was to facilitate the querying a bit by adding to the SQLiteStorage class
```py
conn.row_factory = sqlite3.Row

def query(self, select_string, *args, **kwargs):
    cursor = self.conn.cursor()
    cursor.execute(select_staring, *args, **kwargs)
    return cursor.fetchall()
```

Then the test would look like
```py
def test_log_raiden_run():
    with patch('raiden.storage.sqlite.get_system_spec') as get_speck_mock:
        get_speck_mock.return_value = dict(raiden='1.2.3')
        store = SQLiteStorage(':memory:', None)
    run = store.query('SELECT started_at, raiden_version FROM runs')[0]
    now = datetime.utcnow()
    assert now - timedelta(seconds=2) <= run['started_at'] <= now, f'{run[0]} not right before {now}'
    assert run['raiden_version'] == '1.2.3'
```

Reply:
It depends on how this runs column will be consumed by the client. We already got multiple such utility functions such as can be seen [here](https://github.com/raiden-network/raiden/blob/ed16f6e8791990a94dd57f5b4ebd92d5b069bb7e/raiden/storage/sqlite.py#L158).

But if at the moment this is only consumed by us during debugging and not by the client then there is indeed no need for a utility function.